integrations/Braze/browser.js

```diff
@@ -172,28 +154,103 @@ class Braze {
       "gender",
       "lastname",
       "phone",
-      "facebook",
-      "twitter",
-      "first_name",
-      "last_name",
       "dob",
       "external_id",
       "country",
       "home_city",
       "bio",
-      "gender",
-      "phone",
       "email_subscribe",
       "push_subscribe",
     ];
+    // function set Address
+    function setAddress() {
+      window.braze.getUser().setCountry(address.country);
+      window.braze.getUser().setHomeCity(address.city);
+    }
+    // function set Birthday
+    function setBirthday() {
+      window.braze
+        .getUser()
+        .setDateOfBirth(
+          birthday.getUTCFullYear(),
+          birthday.getUTCMonth() + 1,
+          birthday.getUTCDate()
+        );
+    }
+    // function set Email
+    function setEmail() {
+      window.braze.getUser().setEmail(email);
+    }
+    // function set firstName
+    function setFirstName() {
+      window.braze.getUser().setFirstName(firstname);
+    }
+    // function set gender
+    function setGender(genderName) {
+      window.braze.getUser().setGender(genderName);
+    }
```

Comment:
Why do we need to send the parameter here can't we use the gender?

Reply:
Yes, we have access to gender, but we don't have access to the `this.formatGender()` inside `setGender()` function, so I passed it as a parameter.